src/components/cylc/GScan.vue

```diff
@@ -0,0 +1,102 @@
+<template>
+  <div>
+    <div v-if="workflows && workflows.length > 0">
+      <div v-for="workflow in workflows" :key="workflow.id">
+        <v-list-item :to="`/workflows/${ workflow.name }`">
+          <v-list-item-action>
+            <!-- TODO: the workflows are always running now, but later we will need https://github.com/cylc/cylc-ui/issues/90 adn update the icon displayed here -->
```

Comment:
Note workflows can be running but held.

Reply:
Aaahhh, good point! I added this icon to fill in the space as otherwise the text would be left-aligned without anything before it.

Could we leave this icon for now? I will work on #90 after this, and update GScan to use the new component, which should handle running, held, and.... stopped (?).

Reply:
Should be really easy to implement, the field `workflow.status` field is either `running` or `held`:

```graphql
query {
  workflows{
    status
  }
}
```

Reply:
@kinow - I think you've addressed this, right? (gscan held icon).